### PR TITLE
S3imageuploadandfix #26

### PR DESCRIPTION
**Pull request by davidberard2**
This Branch allowed to fix many issue and complete the image upload feature. All required testing has been completed and seems to be working. Please pay attention to profile_fragment in particular since many changes have been added.

### CODE REVIEW

File: app/src/main/java/Fragments/ProfileFragment.java [hunk 1]
```diff
@@ -68,46 +106,66 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater menuInflater) {
...
                              Bundle savedInstanceState) {
+
         getActivity().getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
 
         view = inflater.inflate(R.layout.fragment_profile, container, false);
 
         name_et = (TextView) view.findViewById(R.id.profile_name);
         email_et = (TextView) view.findViewById(R.id.profile_email);
         phoneNumber_et = (TextView) view.findViewById(R.id.profile_phone_number);
+        ZIP_et = (TextView) view.findViewById(R.id.profile_zip);
```
Comment (LHuss) [inline app/src/main/java/Fragments/ProfileFragment.java:117]: Maybe make this lowercase to remaint consistant with naming. Else, everything looks good.

**Review by LHuss — APPROVED** [review]

**Review by Alex-Hayeur — APPROVED** [review]
The layout changes are appreciated and the code quality is good. Are those commented out lines relevant to this branch or someone else's though? 

Good work either way though.

File: app/src/main/java/Fragments/ProfileFragment.java (hunk 1 above)
Comment (davidberard2) [reply app/src/main/java/Fragments/ProfileFragment.java:117]: Thanks a lot, i'll do so while coding next sprint

**Comment by davidberard2** [comment]
The Commented out lines are relevant. I like to keep them around in case there is need for checks.

**Pull request merged**